src/components/HomePage/Forms/InputWithLabel.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+import {
+  FormGroup, Input, Label,
+} from 'reactstrap';
+import { InputType } from 'reactstrap/types/lib/Input';
+import './Form.scss';
+
+type InputWithLabelProps = {
+  id: string,
+  label: string,
+  name?: string,
+  type?: InputType,
+  placeholder: string,
+  size?: 'lg' | 'sm',
+  readonly?: boolean,
+}
+
+export const InputWithLabel: FC<InputWithLabelProps> = (props) => {
+  const {
+    id, label, name, placeholder, type, size, readonly,
+  } = props;
+  return (
+    <FormGroup>
+      <Label for={id}>
+        {label}
+      </Label>
+      {readonly
+        ? <Input id={id} name={name} type={type} placeholder={placeholder} bsSize={size} readonly="readonly" />
+        : <Input id={id} name={name} type={type} placeholder={placeholder} bsSize={size} />}
```

Comment:
Let's avoid code duplication 
https://stackoverflow.com/questions/31163693/how-do-i-conditionally-add-attributes-to-react-components

Reply:
Thank you. Fixed in commit d0df8953a6498f884fc8f239cf86721b7a4c4d9a